bats/tests/helpers/utils.bash

```diff
@@ -114,6 +114,58 @@ jq_output() {
     return "$status"
 }
 
+# semver returns the first semver version from its first argument (which may be multiple lines).
```

Comment:
Given how hard we pick a version string out of random line noise, I suspect this is more just _version_ and not _semver_ (that is, it's not trying to be semantic). But I guess that's a short-hand for "three-part version numbers" here…